### ISSUE TITLE
Advice adding support for the latest version of reminders

### ISSUE THREAD

**Issue by JonHolman** [picklepete/pyicloud#394]
I'm trying to add support for the latest version of reminders and I've gotten stuck at trying to figure out how to decrypt the text I'm getting back from https://p53-ckdatabasews.icloud.com/database/1/com.apple.reminders/production/private/records/query .

I'm assuming the data I want is in the key TitleDocument, it's value is encrypted and the type is 'ENCRYPTED_BYTES'.

Anyone have experience with these encrypted values?  Have you seen this in other iCloud services?  Any advice would be much appreciated.  Thank you.

## Environment

- pyiCloud release with the issue (`pip show pyicloud`): 
- Last working pyiCloud release (if known): 
- Service causing this issue: 
- Python version (`python -V`): 
- Operating environment (project deps/Docker/Windows/etc.): 

## Traceback/Error logs

```shell

```

## Checklist

- [ ] I've looked informations into the README.
- [ ] I've looked informations into the pyiCloud's code.
- [ ] I've looked informations in Google.

**Comment by ssennettau** [comment]
The values returned with the type `ENCRYPTED_BYTES` appear to be using Base64, which can be decoded using Python's standard `base64` module. Have not attempted to implement it in this codebase, but worth testing. The decoding returns a `byte` object by default, so it must be converted to a string.

```
import base64

base64.b64decode(<VALUE_VARIABLE>).decode("utf-8")
```

**Comment by JonHolman** [comment]
@ssennettau I agree, many of them are, especially in Notes, but in Reminders they appear to be more encrypted or another encoding.

**Comment by bushaev-denis** [comment]
My 5 cents here, we can try to inspect js code from https://icloud.com, i have tried, but without success.
Here is TitleDocument decodes, 0.main.js, find by keywords.
<img width="872" alt="Screenshot 2023-10-07 at 22 38 55" src="https://github.com/picklepete/pyicloud/assets/32451886/3a65427f-84e6-41b5-a87c-6a248e1fa83e">
